### DISCUSSION TITLE
user management, etc.

### DISCUSSION THREAD

**Discussion by skynetradio** [agersant/polaris#195] [Ideas]
Hello everyone

I accidentally discovered Polaris on the web and am very excited.
But I have a question regarding user management.
Because I would like to use Polaris to show my listeners what kind of songs my web station has on offer.

For this it would be helpful to only give the user (guest) rights to view the songs but not to play them!!!
(because of copyrights). Is there or will there be an extended user administration at some point.

1.) How to give access only to certain folders
2.) To give access but to block playback
3.) Fix the layout (colors).
4.) Provide information on quality (FLAC,MP3) and bit rate.

In any case, I really like Polaris.
I would be very happy about an answer.

Greetings from Vienna

**Comment by MaenDera** [comment]
Hey 

I'm actually trying to solve "How to give access only to certain folder". I was able to modify the database in the code and the code for permissions but I couldn't find the web interface to modify it as well, so I'm stuck there😓

Best regards 
Maen

**Reply by agersant** [reply]
Hello! While I'm unlikely to merge PRs adding new features, I'm happy to answer questions that help you implement changes.

The source for the user settings page in the web UI is at: https://github.com/agersant/polaris-web/blob/master/src/components/settings/sections/User.vue
Note that this is in the `polaris-web` repository which is separate from this one (although polaris releases ship both).

Let me know if you need help locating anything else.
